### DISCUSSION TITLE
Section 1 - Bridge

### DISCUSSION THREAD

**Discussion by MIKEX818** [Cyfrin/foundry-full-course-cu#2469] [General]
Hello Everyone, 

I have problem when bridging my Ethereum Sepolia ETH to zkSync testnet ETH. I should have sufficient, but the web just prevent me to bridge to my account... Please check attached screen for details.

Thanks for your help!

<img width="1728" alt="Screenshot 2024-09-14 at 12 55 03 AM" src="https://github.com/user-attachments/assets/809c5a74-5746-4b3b-8766-94baa98f1a95">

**Comment by EngrPips** [comment]
Hello @kaijieshao, It appears to me that you did everything correctly so I am unsure why it is saying insufficient balance when you have some sepolia eth in your wallet. I tried it on my end, and everything works fine.

<img width="1440" alt="Screenshot 2024-09-14 at 08 09 06" src="https://github.com/user-attachments/assets/0f149639-93da-45e3-bc64-0fc1ca2c917f">

Maybe retry some other time.
